packages/app/src/app/pages/Dashboard/Content/CreateNewSandbox/Modal/index.js

```diff
@@ -55,123 +54,73 @@ export default class Modal extends React.PureComponent {
-                <Template
-                  width={Math.floor(paddedWidth / mainTemplatesPerRow - 16)}
-                  key={t.name}
-                  template={t}
-                  selectTemplate={this.selectTemplate}
-                />
-              ))}
-            </Templates>
-          ))}
+  selectTemplate = template => {
+    this.props.createSandbox(template);
+  };
 
-          {serverRows.map((ts, i) => (
-            // eslint-disable-next-line react/no-array-index-key
-            <Templates style={{ fontSize: '.8rem' }} key={i}>
-              {ts.map(t => (
-                <Template
-                  small
-                  width={Math.floor(paddedWidth / templatesPerRow - 16)}
-                  key={t.name}
-                  template={t}
-                  selectTemplate={this.selectTemplate}
-                />
-              ))}
-            </Templates>
-          ))}
+  render() {
+    const { forking = false, closing = false } = this.props;
 
-          <Title>Presets</Title>
-          {typescriptRows.map((ts, i) => (
-            // eslint-disable-next-line react/no-array-index-key
-            <Templates style={{ fontSize: '.8rem' }} key={i}>
-              {ts.map(t => (
-                <Template
-                  small
-                  width={Math.floor(paddedWidth / templatesPerRow - 16)}
-                  key={t.name}
-                  template={t}
-                  selectTemplate={this.selectTemplate}
-                  subtitle={`Using ${t.variantName} template`}
-                />
-              ))}
-            </Templates>
+    const { selectedTab } = this.state;
+    return (
+      <Fragment>
+        <TabContainer>
+          {availableTemplates.map((t, i) => (
+            <Button key={i} selected={selectedTab === i} onClick={() => this.setTab(i)}>
+              {t.name}
+            </Button>
           ))}
-
-          <ImportChoices style={{ marginTop: '1.5rem' }}>
-            <ImportChoice
-              href="/docs/importing#import-from-github"
-              target="_blank"
-            >
-              <GithubLogo /> Import from GitHub
-            </ImportChoice>
-            <ImportChoice
-              href="/docs/importing#export-with-cli"
-              target="_blank"
-            >
-              <TerminalIcon /> Export with CLI
-            </ImportChoice>
-            <ImportChoice href="/docs/importing#define-api" target="_blank">
-              <UploadIcon /> Create with API
-            </ImportChoice>
-          </ImportChoices>
-        </InnerContainer>
-      </Container>
+        </TabContainer>
+        <Container
+          closing={closing}
+          forking={forking}
+          onMouseDown={e => e.preventDefault()}
+        >
+          <InnerContainer forking={forking} closing={closing}>
+            {availableTemplates.map((tab, i) => (
+              <Tab visible={selectedTab === i}>
+                <Templates>
+                  {tab.templates.map(t => (
```

Comment:
kinda seems like cheating to call this `t`. could we please name it `template` like it probably should be?

Reply:
I fixed this and forgot to comment :(